core/src/main/java/com/linecorp/armeria/client/WebClient.java

```diff
@@ -314,6 +315,16 @@ default HttpResponse get(String path) {
         return execute(RequestHeaders.of(HttpMethod.GET, path));
     }
 
+    /**
+     * Sends an HTTP GET request with the specified {@link QueryParams}.
+     */
+    @CheckReturnValue
+    default HttpResponse get(String path, QueryParams params) {
```

Comment:
- How about adding `QueryParams.prepend(String uri)` or something similar instead?
- How about adding `WebClientRequestPreparation.queryParams(QueryParams)` as well?

Reply:
You might want to revert this change and take care of the above comment in a separate PR if you find it more comfortable.

Reply:
Makes sense. Let me handle this in a separate PR.